backend/test/acceptance/api.shoots.spec.js

```diff
@@ -171,10 +169,7 @@ module.exports = function ({ agent, sandbox, k8s, auth }) {
     const actLabels = body.metadata.labels
     const expLabels = metadata.labels
     expect(actLabels).to.eql(expLabels)
-    const actPurpose = body.metadata.annotations['garden.sapcloud.io/purpose']
-    const expPurpose = metadata.annotations['garden.sapcloud.io/purpose']
-    expect(actPurpose).to.equal(expPurpose)
```

Comment:
why is the check removed and not adapted to spec.purpose?

Reply:
Because as purpose is now part of spec
`expect(body.spec).to.eql(spec)`
should to the trick. We could explicitly check for purpose but we also don't do this for other values.

Reply:
will you add
`expect(body.spec).to.eql(spec)`?

Reply:
it is already in

Reply:
ah ok, didn't see it in the diff view